dts/bindings/vendor-prefixes.txt

```diff
@@ -396,6 +396,7 @@ mstar	MStar Semiconductor, Inc. (acquired by MediaTek Inc.)
 mti	Imagination Technologies Ltd. (formerly MIPS Technologies Inc.)
 multi-inno	Multi-Inno Technology Co.,Ltd
 mundoreader	Mundo Reader S.L.
+munic	MUNIC SA
```

Comment:
I don't see any DTS in this PR that would use this?  The hardware IP in question is from ARM, obviously.